DSharpPlus/Entities/Channel/Message/DiscordAttachment.cs

```diff
@@ -72,6 +72,12 @@ public class DiscordAttachment : SnowflakeObject
         [JsonProperty("width", NullValueHandling = NullValueHandling.Ignore)]
         public int? Width { get; internal set; }
 
+        /// <summary>
+        /// Gets whether the attachemnt is ephemeral.
+        /// </summary>
+        [JsonProperty("ephemeral", NullValueHandling = NullValueHandling.Ignore)]
+        public bool Ephemeral { get; internal set; }
```

Comment:
```suggestion
        public bool? Ephemeral { get; internal set; }
```
This should be nullable.